docs/user-guides/microservice.md

```diff
@@ -40,4 +43,8 @@ service_description:
     project: X1
   instance_properties:
     nodeIP: 192.168.0.111
+  paths:
+  - path: /rest/demoservice
+    property:
+      checksession: true
```

Comment:
what that checksession means

Reply:
checksession just a demo  property, SC will store it